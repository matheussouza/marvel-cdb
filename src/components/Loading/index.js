import React from 'react';
import loadingImage from '../../assets/img/loading.png';

const Loading = () => (
  <div className="loading-container">
    <img src={loadingImage} className="mdb-loading" alt="Loading" />
  </div>
);

export default Loading;
