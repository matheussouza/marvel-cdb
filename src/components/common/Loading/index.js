import React from 'react';
import loadingImage from '../../../assets/img/loading.png';

export default () => (
  <div className="loading-container">
    <img src={loadingImage} className="mdb-loading" alt="Loading" />
  </div>
);
