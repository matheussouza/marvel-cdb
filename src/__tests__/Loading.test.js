import React from 'react';
import { render, screen } from '@testing-library/react'
import Loading from '../components/Loading';

describe('Loading component', () => {

  it('should exist', () => {
    expect(Loading).toMatchSnapshot();
  });

  it('should contain an image', () => {
    render(<Loading />);

    expect(screen.getByRole('img')).toExist();
  });

});