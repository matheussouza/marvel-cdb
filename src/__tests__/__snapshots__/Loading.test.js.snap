// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component should exist 1`] = `[Function]`;
